_data/governments.yml

```diff
@@ -464,6 +464,7 @@ Russia:
   - cikrf
   - kznru
   - moscow-technologies
+  - ghalghai
```

Comment:
I think when the CI runs, it will probably complain about the sorting of the list. Might want to move it up before `kznru`

Reply:
Thank you!